docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,651 @@
+
+Creates a new Jira issue with specified details.
+
+#### Configuration
+
+| Name                         | Type     | Required | Description                                                                                                 |
+| ---------------------------- | -------- | -------- | ----------------------------------------------------------------------------------------------------------- |
+| `createIssue.projectKey`     | `string` | Y        | The key of the Jira project where the issue will be created.                                                |
+| `createIssue.summary`        | `string` | Y        | The summary or title of the issue.                                                                          |
+| `createIssue.description`    | `string` | N        | The description of the issue. Supports markdown formatting.                                                 |
+| `createIssue.adfDescription` | `object` | N        | ADF (Atlassian Document Format) content for complex formatting. Alternative to `description`.               |
+| `createIssue.issueType`      | `string` | N        | The type of issue to create (e.g., 'Bug', 'Task', 'Story').                                                 |
+| `createIssue.assigneeEmail`  | `string` | N        | Email of the user to assign the issue to.                                                                   |
+| `createIssue.labels`         | `array`  | N        | Labels to add to the issue for categorization.                                                              |
+| `createIssue.customFields`   | `object` | N        | Custom fields to set. Keys should match Jira custom field IDs.                                              |
+| `createIssue.issueAlias`     | `string` | N        | Override for the freight metadata key used to reference the created issue id. Defaults to `jira-issue-key`. |
+
+#### Output
+
+| Name  | Type     | Description                                           |
+| ----- | -------- | ----------------------------------------------------- |
+| `key` | `string` | The key/id of the created Jira issue (e.g., EXT-123). |
+
+#### Example
+
+This example creates a new Jira issue to track a deployment, assigns it to a team member, and adds relevant labels.
+
+```yaml
+steps:
+  - uses: jira
+    as: create-deployment-issue
+    config:
+      credentials:
+        secretName: jira-credentials
+      createIssue:
+        projectKey: DEPLOY
+        summary: "Deploy ${{ imageFrom(vars.imageRepo).Tag }} to ${{ ctx.stage.name }}"
+        description: "Deploying ${{ imageFrom(vars.imageRepo).RepoURL }}:${{ imageFrom(vars.imageRepo).Tag }} to ${{ ctx.stage.name }} environment. Promotion ID: ${{ ctx.promotion.name }}. Freight: ${{ ctx.targetFreight.name }}."
+        issueType: Task
+        assigneeEmail: devops@company.com
+        labels:
+          - deployment
+          - "${{ ctx.stage.name }}"
+          - "release-${{ imageFrom(vars.imageRepo).Tag }}"
```

Comment:
In code and docs, Kargo has tried to standardize on this convention WRT to YAML lists:

```suggestion
        labels:
        - deployment
        - "${{ ctx.stage.name }}"
        - "release-${{ imageFrom(vars.imageRepo).Tag }}"
```

Reply:
May apply in other spots.